client/src/components/UtcDate.vue

```diff
@@ -33,7 +33,7 @@ export default {
             return moment.utc(this.formattedDate).format();
         },
         formattedDate: function () {
-            if (this.customFormat) {
+            if (this.customFormat !== undefined) {
```

Comment:
```(this.customFormat)``` is kinda concise and does the same IMHO... sorry

Reply:
It's similar, resolving `this.customFormat`'s truthiness, but especially in javascript I lean towards preferring explicit, strict comparisons when you know what the inputs/types are expected to be.  In any situation where this is defined (`!== undefined`), someone is trying to do something with the formatting here and we want it attempted so it, at minimum, blows up if it's not right.